### PR TITLE
Iterate Account and Transaction API.

### PR DESCRIPTION
**Pull request by nkristianto**
Iterate and update account and transaction api from ACB proto.
Add ApprovePayment and TFA on transaction API. Add CheckAccount,
UpdateAccountStatus, and GetAccountStatus on account API.
Also update comment on Branch API. The only changes is the gunk file.

DONE [OB-197]
DONE [OB-198]

### CODE REVIEW

**Comment by nkristianto** [comment]
@mvdan , if you don't mind, please take a look of this PR. Heikki and Cuong is on leave today. Thank you

File: v1/accounts/accounts.gunk [hunk 1]
```diff
@@ -248,12 +248,65 @@ type UpdateAccountRequest struct {
...
 
+// UpdateAccountStatusRequest is the request envelope to update
+// the account status.
+type UpdateAccountStatusRequest struct{
+	// AccountID is the account unique identifier.
+	AccountID string `pb:"1" json:"account_id"`
+
+	// Status it the status to be updated
+	// TODO: Change the string parameters into type
+	// once the status of the account is identified.
```
Comment (mvdan) [inline v1/accounts/accounts.gunk]: this TODO will show up as part of the docs; move it to a separate comment.

File: v1/accounts/accounts.gunk [hunk 2]
```diff
@@ -248,12 +248,65 @@ type UpdateAccountRequest struct {
...
+
 // DeleteAccountRequest is the request envelope to delete an account.
 type DeleteAccountRequest struct {
-	// AccountID is the unique identifier of the account to delete.
+	// AccountID is the account unique identifier.
 	AccountID string `pb:"1" json:"account_id"`
 }
 
+// CheckAccountRequest is the request to get the information
+// whether the account exist or not.
```
Comment (mvdan) [inline v1/accounts/accounts.gunk]: exists

File: v1/accounts/accounts.gunk [hunk 3]
```diff
@@ -248,12 +248,65 @@ type UpdateAccountRequest struct {
...
+type CheckAccountRequest struct{
+	// AccountID is the account unique identifier.
+	AccountID string `pb:"1" json:"account_id"`
+}
+
+// CheckAccountResponse is minimum set of information about
```
Comment (mvdan) [inline v1/accounts/accounts.gunk]: the minimum

File: v1/accounts/accounts.gunk [hunk 4]
```diff
@@ -248,12 +248,65 @@ type UpdateAccountRequest struct {
...
+// account existension.
```
Comment (mvdan) [inline v1/accounts/accounts.gunk]: existension isn't a word - what are you trying to say?

**Review by mvdan — COMMENTED** [review]
Also, please use one commit.

File: v1/accounts/accounts.gunk [hunk 5]
```diff
@@ -248,12 +248,65 @@ type UpdateAccountRequest struct {
...
+type CheckAccountResponse struct{
+	// AccountID is the account unique identifier.
+	AccountID string `pb:"1" json:"account_id"`
+
+	// BankCode is the code that related to bank.
```
Comment (kofoworola) [inline v1/accounts/accounts.gunk]: The code that is* related to the bank

File: v1/transactions/transactions.gunk [hunk 6]
```diff
@@ -196,6 +196,63 @@ type CreateTransactionResponse struct {
...
+	Approve bool `pb:"2" json:"approve"`
+
+	// TFAType is type to receive OTP authentication code.
+	TFAType TFAType `pb:"3" json:"tfa_type"`
+}
+
+// ApprovePaymentResponse response for approve payment request.
+type ApprovePaymentResponse struct{
+	// AuthorizationID is the executable code is obtained from
+	// the payment feedback result
```
Comment (kofoworola) [inline v1/transactions/transactions.gunk:228]: I kinda don't understand this line

**Review by kofoworola — COMMENTED** [review]
Minor documentation comments. Asides those. The whole process looks good and understandable

File: v1/transactions/transactions.gunk (hunk 6 above)
Comment (nkristianto) [reply v1/transactions/transactions.gunk:228]: i got this from ACB proto definition, not sure what that mean. I will ask Cuong later on.

File: v1/accounts/accounts.gunk (hunk 1 above)
Comment (nkristianto) [reply v1/accounts/accounts.gunk]: I will move this to implementation side. I'll remove the TODO for now

**Comment by nkristianto** [comment]
PTAL @mvdan @kofoworola

**Review by mvdan — APPROVED** [review]

**Pull request merged**